code/src/index.css

```diff
@@ -1,13 +1,84 @@
 body {
+  font-family: 'Noto Sans Mono', monospace;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(255, 212, 233);
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: 'Noto Sans Mono', monospace;
 }
+
+.outerWrapper {
+  /*border: 2px solid green;*/
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.innerWrapper {
+  /* border: 2px solid rgb(128, 0, 0); */
+  width: 50%;
+}
+
+.messageForm {
+  border: 1px solid black;
+  background-color: rgb(227, 227, 227);
+  margin: 30px;
+  padding-left: 40px;
+  padding-bottom: 30px;
+  box-shadow: 7px 7px;
+}
+
+.formBtn{
+  font-family: 'Noto Sans Mono', monospace;
+  background-color: rgb(241, 198, 198);
+  border: none;
+  margin-top: 10px;
+  border-radius: 20px;
+  padding: 10px;
+}
+
+.formBtn:hover {
+  border: 1px solid gray;
+  cursor: pointer;
+}
+
+.textArea {
+  width: 90%;
+  height: 80px;
+}
+
+.singleMessage {
+  background-color: rgb(250, 250, 250);
+  border: 1px solid black;
+  margin: 30px;
+  padding: 10px;
+  box-shadow: 7px 7px;
+  font-size: 14px;
+}
+
+.likeAndTime{ 
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.btnAndCounter {
+  display: flex;
+  flex-direction: row;
+}
+
+.likeBtn {
+  margin-right: 10px;
+  padding: 15px;
+  line-height: 0%;
+  border-radius: 50%;
+  border: none;
+}
+
+.likeBtn:hover {
+  border: 1px solid gray;
+  cursor: pointer;
+}
```

Comment:
Also nice to add a hover effect here! Goes very well with the rest of the design here too :)